project/src/components/gallery/gallery.tsx

```diff
@@ -0,0 +1,28 @@
+import {MAX_OFFER_IMAGES} from '../../const';
+
+type GalleryProps = {
+  images: string[];
+}
+
+function Gallery(props: GalleryProps): JSX.Element {
+  console.log(props);
+  console.log(props.images);
+
+  const imageItem = props.images.slice(0, MAX_OFFER_IMAGES).map((item) => (
```

Comment:
Раз здесь коллекция то 
```suggestion
  const imageItems = props.images.slice(0, MAX_OFFER_IMAGES).map((item) => (
```
Лайк за константу